### PR TITLE
Removed method reference to previous lesson

### PR DESCRIPTION
**Pull request by martink-rsa**
Referring to numbered list item 3 under heading Strings which states as follows:

3. Vocabulary time: a __method__ is a bit of functionality that is built into the language or into specific data types.  In the previous lesson you learned a few methods that can be used on strings such as `indexOf` and `search`.  An exhaustive list of methods that can be used on strings can be found [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String).

The second sentence is incorrect as it states that methods were discussed in the previous lesson, however this is not true. Methods are only introduced in the numbered list item 2 which precedes this item. I believe it is most likely that there was a hyperlink in the previous lesson where methods are touched on, however this was removed at a later stage.

This sentence could create confusion for the student who may end up revising all prior work to see where they had missed the information on methods, only to discover that there was no information provided.

One can change the sentence to read that methods were introduced in the preceding numbered point, however this could lead to the same issue if the preceding numbered list item is altered and the method information removed.

It makes more sense to me to remove the sentence that refers to where the individual had found the information on methods: 
"In the previous lesson you learned a few methods that can be used on strings such as `indexOf` and `search`. "

The paragraph would then read as follows, as reflected by the Push request:
"3. Vocabulary time: a __method__ is a bit of functionality that is built into the language or into specific data types. An exhaustive list of methods that can be used on strings can be found [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String).

This is a PR template. If you are adding a solution link to the curriculum, leave this as is. If not, delete it and write the message you wish.

Thank you,

The Odin Project team.

### CODE REVIEW

File: web_development_101/javascript_basics/fundamentals-2.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Depending on what kind of work you're doing, you might end up working more with
 
 1. Read and code along with [yet another MDN tutorial](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/Strings) on the topic.
 2. Go through [this lesson](https://www.w3schools.com/js/js_string_methods.asp) to learn a bit more about what you can do with strings... be sure to do the exercises at the end!
-3. Vocabulary time: a __method__ is a bit of functionality that is built into the language or into specific data types.  In the previous lesson you learned a few methods that can be used on strings such as `indexOf` and `search`.  An exhaustive list of methods that can be used on strings can be found [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String).
+3. Vocabulary time: a __method__ is a bit of functionality that is built into the language or into specific data types.  An exhaustive list of methods that can be used on strings can be found [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String).
```
Comment (CouchofTomato) [inline web_development_101/javascript_basics/fundamentals-2.md]: I'm not sure if it makes sense to link to an exhaustive list of methods for strings without any context.

**Comment by leila-alderman** [comment]
@martink-rsa Perhaps the intention here is unclear, but as you mentioned, `the previous lesson` is referring to the previous bullet point, which links to https://www.w3schools.com/js/js_string_methods.asp, a lesson on different string methods, including the two explicitly mentioned there. 

I think that this can fairly simply be changed to state 
`In [the previous exercise](https://www.w3schools.com/js/js_string_methods.asp),`

**Comment by martink-rsa** [comment]
> @martink-rsa Perhaps the intention here is unclear, but as you mentioned, `the previous lesson` is referring to the previous bullet point, which links to https://www.w3schools.com/js/js_string_methods.asp, a lesson on different string methods, including the two explicitly mentioned there.
> 
> I think that this can fairly simply be changed to state
> `In [the previous exercise](https://www.w3schools.com/js/js_string_methods.asp),`

I believe that this would be the simplest way to handle it and I think you've nailed the problem regarding it. I didn't consider that it was "lesson" that was obfuscating the matter. 

An alternative is to mention where the lesson is from so the student is aware that it is not a TOP lesson being referred to. This can be seen in the TOP Fundamentals Part 1 lesson directly before the one being discussed: https://www.theodinproject.com/courses/web-development-101/lessons/fundamentals-part-1

You will see in the lesson above that it states "This W3Schools lesson..." as opposed to simply "This lesson...". 

This would result in:

> 3. Vocabulary time: a __method__ is ...  In the previous W3Schools lesson you learned a few methods that can...

**Comment by leila-alderman** [comment]
Yep, I see what you're saying, and my attempt at making that clearer in my suggestion was to add a link back to that W3 lesson page for the text `the previous exercise`. Do you think that this would resolve the confusion?

`In [the previous W3Schools exercise](https://www.w3schools.com/js/js_string_methods.asp), ...`

**Comment by martink-rsa** [comment]
> 
> 
> Yep, I see what you're saying, and my attempt at making that clearer in my suggestion was to add a link back to that W3 lesson page for the text `the previous exercise`. Do you think that this would resolve the confusion?
> 
> `In [the previous W3Schools exercise](https://www.w3schools.com/js/js_string_methods.asp), ...`

I believe your suggestion will resolve the matter.

**Pull request merged**